doc/nrf/known_issues.rst

```diff
@@ -349,7 +447,7 @@ DESK-978: Directed advertising issues with SoftDevice Link Layer
 
```

Comment:
I just spotted that the note right above isn't indented correctly, and thus shows up when it should be filtered out. No need to change for the other PR imo, but you could fix it here for master.

Reply:
Fixed